web/src/lib/components/faces-page/merge-suggestion-modal.svelte

```diff
@@ -8,27 +8,57 @@
   import { mdiArrowLeft, mdiClose, mdiMerge } from '@mdi/js';
   import Icon from '$lib/components/elements/icon.svelte';
 
-  const dispatch = createEventDispatcher<{
-    reject: void;
-    confirm: [PersonResponseDto, PersonResponseDto];
-    close: void;
-  }>();
+  import { assetViewingStore } from '$lib/stores/asset-viewing.store';
 
   export let personMerge1: PersonResponseDto;
   export let personMerge2: PersonResponseDto;
   export let potentialMergePeople: PersonResponseDto[];
 
+  let { isViewing: showAssetViewer } = assetViewingStore;
   let choosePersonToMerge = false;
+  let changeFocus = false;
+  let buttonNo: HTMLButtonElement;
+  let buttonYes: HTMLButtonElement;
 
   const title = personMerge2.name;
 
+  const dispatch = createEventDispatcher<{
+    reject: void;
+    confirm: [PersonResponseDto, PersonResponseDto];
+    close: void;
+  }>();
+
+  const handleKeyboardPress = (event: KeyboardEvent) => {
+    if (!$showAssetViewer) {
+      event.stopPropagation();
+      switch (event.key) {
+        case 'Tab':
```

Comment:
Tabbing through focusable elements is something that is built into the browser. We don't need to re-invent the wheel here.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

Reply:
Yeah I know but I couldn't find a way to limit the focus to the suggested people only.

Reply:
tabindex=-1 tells the browser to skip it.

Reply:
Yeah I tried it but the issue is that you want to navigate only through the suggested people, not all the buttons on the page. Do you have any ideas :thinking: ?

Reply:
Does using an actual browser model still let you focus on background elements?

Reply:
What's an actual browser model ?

Reply:
Mmmh, just tried it, does it work only for dialogs ?

Reply:
I don't think we want dialogs to display suggested people :sweat_smile: